### DISCUSSION TITLE
How can I test if it works

### DISCUSSION THREAD

**Discussion by El-Hondo** [wawawario2/text-generation-webui#18] [Q&A]
I don't this extension to work perfectly. But I could not come up with an example where the AI could remember anything from a past conversation. Does a Clear history have any effects on the memories?
Can someoen provide a simple test were I very it works?

**Comment by JoureGameDev** [comment]
It doesn't have  a 100% success rate but what I did was to give it my name. I had a long convo and after a while I asked it my name and it "remembered" I shut down the model and restarted it. 

I asked it again if it knew my name and it answered correctly. 

I then shut down my pc and rebooted it. Again I asked it my name and it again answered correctly. After that I started talking again and it got it wrong. I mentioned that fact and it got it right again after that. 

So yeah, I think it queries a single fact or "memory" and tries to get it right.

Anyway, that's how I "tested" it.

Hope that helps.
